### ISSUE TITLE
Support for Vim signs

### ISSUE THREAD

**Issue by alexbooker** [flowtype/vim-flow#22]
What do you think about using [Signs](http://vimdoc.sourceforge.net/htmldoc/sign.html) to indicate erroneous lines?

**Comment by orlando** [comment]
@alexbooker is this easy to integrate?. window split is a pain on a macbook pro 13 inch

**Comment by alexbooker** [comment]
@orlando I'm never tried so I don't know.

I use Atom when I use Flow nowadays.

**Comment by orlando** [comment]
@alexbooker cool, I'll give a try next week to see if I can come up with something.

Thanks!

**Comment by alexbooker** [comment]
@orlando Sounds awesome :smile: 

I would just be a bit wary of the fact that the maintainers of this repository _might_ not want to merge such a feature.

**Comment by orlando** [comment]
@alexbooker I was about to start work on this, but looks like [Syntastic](https://github.com/scrooloose/syntastic) already [supports it](https://github.com/scrooloose/syntastic/blob/fea5d5240a45ea412350bd70fb8af6708925ede5/syntax_checkers/javascript/flow.vim). Just add it to your javascript checkers (`let g:syntastic_javascript_checkers = ['eslint', 'flow'])` and that's it.

So I'm closing this issue

**Comment by andreypopp** [comment]
Same for neomake, it works well with flow.

**Comment by alexbooker** [comment]
Since I posted this **two** months ago I haven't really used **`vim-flow`**. Instead I use Atom so I'm not in a good position to comment.

**Comment by ryyppy** [comment]
Neomake does this really really well!

**Comment by rafaelrinaldi** [comment]
@ryyppy That's true, I'm delegating this to Neomake and it _just works_. Do you think is even worth having this plugin handling both linting and syntax highlighting? Maybe it's better to just take care of the ladder then suggesting other tools like syntastic and Neomake for linting, since they're out there for a longer term therefore more reliable options.

**Comment by orlando** [comment]
@ryyppy how do you set up neovim to do that. I want to switch to Neovim starting next month and this is one of the features I want to get in place from the beginning.

**Comment by rafaelrinaldi** [comment]
@orlando Neomake comes with Flow support. You just have to make sure you setup `flow` as a maker.

**Comment by orlando** [comment]
@rafaelrinaldi

> You just have to make sure you setup flow as a maker.

can you explain that part please?. I didn't get it

**Comment by rafaelrinaldi** [comment]
@orlando Something like this should do it:

```viml
" Disable Facebook's Flow Vim linter to avoid collisions
let g:flow#enable = 0

" Enable Flow maker
let g:neomake_javascript_enabled_makers = ['flow']

" Lint whenever saving a file
au BufWritePost * Neomake
```

**Comment by rafaelrinaldi** [comment]
@orlando I just wrote a more detailed explanation [here](https://github.com/rafaelrinaldi/til/blob/master/vim/linting-with-flow.md).

**Comment by orlando** [comment]
@rafaelrinaldi awesome. Great idea to have a **til** repo

Cheers!
